### DISCUSSION TITLE
Instant Slope (Feature request)

### DISCUSSION THREAD

**Discussion by Pilou8** [bailuk/AAT#149] [Ideas]
Hello,

Since I don't have any bike comptuer, I have been using AAT for about two years now when road and monuntain biking.
But one thing miss me a lot : I would like to know the instant slope percentage.
Would it be possible for AAT to show it, based both on GPS and accelerometer information?

That would be great, as I actually dont know another free app offering this feature!

Many Thanks !

**Comment by frasty** [comment]
I am looking for such a feature for so long. It would be great if AAT could implement instant slope recording from the accelerometer sensor as I believe it would be much more accurate than the GPS derived slope.

**Comment by watchingJu** [comment]
Might be a bit of work, but there exists a master thesis on this calculation:
https://lup.lub.lu.se/luur/download?func=downloadFile&recordOId=8847441&fileOId=8859296

Refered here:
https://math.stackexchange.com/questions/2384957/reading-ground-road-slope-from-accelerometer-data#
